packages/move/sources/vesting.move

```diff
@@ -89,6 +111,10 @@ module deployment_addr::vesting {
 
     // ================================= Entry Functions ================================= //
 
+    // FIX: the following global admin functions would not be needed if each VestingStream is a member of Object, since each Object has an owner
+
+    // - - - ADMIN FUNCTIONS - - -
+
```

Comment:
Indeed, can be removed in favor of transfering ownership of the Object for each VestingStream.